src/pages/ReimbursementAccount/RequestorStep.js

```diff
@@ -155,24 +155,30 @@ class RequestorStep extends React.Component {
                     onCloseButtonPress={Navigation.dismissModal}
                 />
                 {this.props.achData.useOnfido && this.props.achData.sdkToken && !this.state.isOnfidoSetupComplete ? (
-                    <Onfido
-                        sdkToken={this.props.achData.sdkToken}
-                        onUserExit={() => {
+                    <ScrollView
+                        contentContainerStyle={[
+                            styles.mnh100,
```

Comment:
Why do we need this style? Could you please inline this?

Reply:
Both works, will change it.